build/static/style.css

```diff
@@ -0,0 +1,21 @@
+.workload-summary-cell{
+  max-width: 40px;
+  overflow: hidden;
+}
+
+.red, .green{
+  color: white;
+}
+.red{
+  background-color: red;
+}
+.green{
+  background-color: green;
+}
+.yellow{
+  background-color: yellow;
+}
```

Comment:
this is an old way of doing styling declaratively. It also hurts my soul a tiny bit but i'm okay with it. These are also not the bcgov colors

Reply:
Another thing that should get resolved by changes I'm about to work on.